zh.js

```diff
@@ -210,10 +210,10 @@ settings.interface.hideNames.all 全部
 settings.interface.nametagStyle 名牌样式
 settings.interface.nametagOpac 名牌不透明度
 settings.interface.nametagScale 名牌显示比例
-settings.interface.nametagStyle.health 仅限血条
-settings.interface.nametagStyle.name 仅限名称
-settings.interface.nametagStyle.nameLvl 仅限名称和等级
-settings.interface.nametagStyle.all 全部
+settings.interface.nametagStyle.health 只显示血条
```

Comment:
Again keep it simple as in en.js it is "Health only" and not "Only displays Health". (Same for line 214-216)